.github/workflows/docs/build-docs

```diff
@@ -10,8 +10,9 @@ import sys
 
 DOCS_DIR = Path(sys.argv[0]).absolute().parent
 MODULES_DIR = DOCS_DIR.parent.parent.parent
-PYTKET_DOCS_LINK = "https://cqcl.github.io/tket/pytket/api/index.html"
-PYTKET_EX_DOCS_LINK = "https://cqcl.github.io/pytket-extensions/api/index.html"
+TKET_WEBSITE_LINK = "https://tket.quantinuum.com/"
+PYTKET_DOCS_LINK = "https://tket.quantinuum.com/api-docs/"
+PYTKET_EX_DOCS_LINK = "https://tket.quantinuum.com/api-docs/extensions.html"
```

Comment:
Wrong URL?

Reply:
You mean this one? https://tket.quantinuum.com/api-docs/extensions.html

That is correct, but not working yet. It will work after the next pytket release.

Reply:
The `https://tket.quantinuum.com/api-docs/extensions.html` page isn't deployed yet but should be after the next pytket release.

Reply:
@CalMacCQ can you confirm this?

Reply:
Yes, the the URL you gave is correct. The extensions index is now part of the API docs. This link should work when the tket-site is redeployed after the next pytket release.